test/SourceKit/lit.local.cfg

```diff
@@ -15,6 +15,7 @@ elif 'swift_evolve' in config.available_features:
 else:
     sk_path_sanitize = os.path.join(os.path.dirname(__file__), 'Inputs', 'sourcekitd_path_sanitize.py')
 
+    config.substitutions.append( ('%sourcekitd-test_plain', config.sourcekitd_test) )
```

Comment:
I did not find a way to make a resolution of the `#colorLiteral` init work without the `-sdk %sdk` flag. Without 
the `%sourcekitd-test_plain` a temporary module cache is used where the AppKit module cache needs to be rebuilt first. This takes some time and sourcekit timeouts. Is there a standard/better way to solve this? 🤔

Reply:
I haven’t tried this myself but in complete_literal.swift we do the following, which, I think, sets the type to be used by `#colorLiteral` so we don’t need to depend on AppKit/UIKit (which we wouldn’t be able to do on Linux and Windows anyway because they don’t have AppKit/UIKit) Would you mind giving it a try? If it doesn’t work, let me know and I’ll investigate further.

https://github.com/apple/swift/blob/05919751b4c9b86f5188947d2609d32955d1b3ab/test/IDE/complete_literal.swift#L54-L58

Reply:
Great that works, thank you! 👍 I removed the `sourcekitd-test_plain` along with the now unnecessary open request.